client/src/ui-components/ClientOrdersComponent/ClientOrderTableRow.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import "./ClientOrderTableRow.css";
+function ClientOrderTableRow(props) {
+  const [status, setStatus] = useState(props.order.status);
+
+  const handleSubmit = () => {
+    if (status === "WAITING") {
+      setStatus("COMPLETED");
+    } else if (status === "COMPLETED") {
+      setStatus("WAITING");
```

Comment:
If the status is DONE (meaning it was given to the client) it cannot return to a previous state
Also, a PopUp should be shown when "change status" is clicked to ask for confirmation